src/app/shared/components/tab-card/style.scss

```diff
@@ -36,6 +36,8 @@
 
     .mat-ink-bar {
       top: 0;
+      transition-duration: 200ms;
+      //transition: 200ms cubic-bezier(0.35, 0, 0.25, 1);
```

Comment:
Please remember to remove the commented part before merging.

Reply:
damn it, happened again. sorry. fixed

Reply:
np 🙂